app/controllers/partners_controller.rb

```diff
@@ -5,7 +5,8 @@ class PartnersController < ApplicationController
   include Importable
 
   def index
-    @partners = current_organization.partners.order(:name)
+    @unfiltered_partners_for_statuses = Partner.where(organization: current_organization)
```

Comment:
I'm not how I like doing Partner.where vs current_organization.partners. Is there a benefit to not grabbing the partners off of the current_organization?

Reply:
Mostly I feel we did this before and it bit us in the butt.

Reply:
oh I guess it could be the old way, just without filter params